plugins/feeds/public/CertPaIT.py

```diff
@@ -0,0 +1,35 @@
+import logging
+from datetime import datetime, timedelta
+from core.observables import Hash
+from core.feed import Feed
+from core.errors import ObservableValidationError
+
+class CertPaIt(Feed):
+
+    default_values = {
+        "frequency": timedelta(minutes=30),
+        "name": "CertPaIT",
+        "source" : "https://infosec.cert-pa.it/analyze/submission.rss",
+        "description": "This feed contains data from infosec.cert-pa.it",
+    }
+
+    def update(self):
+        for item in self.update_xml('item', ["title", "link"]):
+            self.analyze(item)
+
+    def analyze(self, item):
+        md5 = item['title'].replace("MD5: ", "")
+        context = {}
```

Comment:
There's a bunch of other information you can extract from the items `description` field:

```xml
<description>
<![CDATA[
<p>Filename: <b>x86_32</b><br>Filetype: ELF 32-bit LSB executable, Intel 80386, version 1 (GNU/Linux), statically linked, stripped</p> <ul> <li>IsDLL: No</li> <li>Packers: No</li> <li>AntiDBG: No</li> <li>AntiVM: No</li> <li>Signed: No</li> <li>XOR: No</li> </ul>
]]>
</description>
```

I think this would be very useful to have in the `context` dict!

Reply:
ok adding :)